src/PIL/ImageFilter.py

```diff
@@ -442,6 +442,15 @@ def transform(self, callback, with_normals=False, channels=None,
                           target_mode=target_mode or self.mode,
                           _copy_table=False)
 
+    def __repr__(self):
+        r = ["{0} from {1}".format(self.__class__.__name__,
+                                   self.table.__class__.__name__)]
+        r.append("size={0}x{1}x{2}".format(*self.size))
+        r.append("channels={0}".format(self.channels))
```

Comment:
Minor styling issue - would it not be clearer to include all three items in the initial declaration of the array, rather than using two `append` statements?

Reply:
Done